template/build.cmd

```diff
@@ -0,0 +1,17 @@
+@ECHO off
+SETLOCAL
+
+SET BUILD_FOLDER=.build
+
+SET BOOTSTRAP_FILE_REMOTE=https://raw.githubusercontent.com/aspnet/KoreBuild/victorhu/init/bootstrap/build.cmd
+SET BUILD_FILE=%BUILD_FOLDER%\build.cmd
+
+SET REPO_FOLDER=%~dp0
+CD "%REPO_FOLDER%"
```

Comment:
Don't bury the lead. Expect to see these two lines at the top of the file -- just after `SETLOCAL`.